docs/docker-machine-drivers.md

```diff
@@ -1,13 +1,9 @@
 # Docker Machine driver installation
 
-Minishift uses Docker Machine to manage the OpenShift VM, so it benefits from the
-driver plugin architecture that Docker Machine uses to provide a consistent way to
-manage various VM providers. Minishift embeds VirtualBox and VMware Fusion drivers
-so there are no additional steps to use them. However, other drivers require an
-extra binary to be present on the _PATH_.
+Minishift uses Docker Machine and its driver plugin architecture to provide a consistent way to manage the OpenShift VM and other VM providers. Minishift embeds VirtualBox and VMware Fusion drivers
```

Comment:
@hferentschik, how about this, would this be factually correct: Minishift uses Docker Machine and its driver plugin architecture to provide a consistent way to manage the OpenShift VM. ?